html/theme.css

```diff
@@ -86,3 +86,19 @@ a {
 /* p {
     font-size: 16px;
 } */
+
+body table:nth-child(1)[width="100%"] {
+    display: none;
+}
+
+h1, h2 {
+    text-align: center;
+}
+
+img {
+    display: none;
+}
+
+a ~ div.header {
```

Comment:
Is it possible to make the header look better rather than disappear? If I want to know which package the function belong to, then I have to scroll to bottom and see the line `[Package ??? version 4.0.3 Index]`.

Reply:
Maybe it's possible to move the package name to the top right corner and not show the "R Documentation" label? I'm not very good with CSS but I'll see what I can do tomorrow

Reply:
Ok that was easier than expected...
I moved the package name to the top right corner to make it easier to "find" the beginning of the description when reading.

Reply:
Already looks better. Thanks!